spiderable_server.js

```diff
@@ -16,6 +16,13 @@ var crypto = Npm.require('crypto');
 Spiderable.userAgentRegExps = [
     /^facebookexternalhit/i, /^linkedinbot/i, /^twitterbot/i];
```

Comment:
@jazeee Agree to `bingbot`, also while we was working on our app, we found next bots as most common in the web, may be you want to add any of them also?:
  yandex
  Google-Structured-Data-Testing-Tool
  Yahoo
  MJ12bot
  TweetmemeBot
  visionutils
  Baiduspider
  Googlebot
  Mail.RU_Bot
  bingbot
  yahoo
  bingbot
  AhrefsBot
  vkShare
  SiteLockSpider